### ISSUE TITLE
PowerPC repository has moved

### ISSUE THREAD

**Issue by exdead01** [neagix/wii-linux-ngx#18]
The PowerPC Jessie repository has moved from deb.debian.org/debian/ to archive.debain.org/debian/

**Comment by emukidid** [comment]
I believe you mean: archive.debian.org/debian/

**Comment by exdead01** [comment]
Yeah, sorry for the typo.

**Comment by SpazmasterYT** [comment]
Ah, finally. I was wondering why I was getting 404's

**Comment by mehlon** [comment]
According to https://wiki.debian.org/SourcesList#Configuring_Apt_Sources : 
Change the `/etc/apt/sources.list` file to:
```deb http://archive.debian.org/debian/ jessie main```
then run `apt update`

**Comment by naruse** [comment]
This is great and all but unfortunately as of now Apr 17th 2023 its complaining about GPG errors 
```
W: GPG error: http://archive.debian.org jessie Release: The following signatures were invalid: KEYEXPIRED 1587841717
```
I tried this https://unix.stackexchange.com/questions/598344/debian-8-jessie-keyexpired-1587841717 and no luck.

Changed the `/etc/apt/sources.list` to `deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main` (https://unix.stackexchange.com/questions/508724/failed-to-fetch-jessie-backports-repository)

I even tried to do the apt-get update with these args `apt-get -oAcquire::Check-Valid-Until=no -oAcquire::AllowInsecureRepositories=true update` and still nope

Has anyone else stumbled upon this?, I have got several key not found and fixed it with this https://chrisjean.com/fix-apt-get-update-the-following-signatures-couldnt-be-verified-because-the-public-key-is-not-available/

Any thoughts wrt this nowadays 2023 :/?, I am not expecting to get the latest versions of everything since this is for PowerPC architecture and PowerPC was supported until Debian 8, but I would expect to at least be able to install certain packages like emacs and others to make this OS usable :(

**Comment by omerien** [comment]
Spending some time on Debian mailing lists, here is a solution that might work :"no public key available" on apt-get updateunix.stackexchange.comIf you’re asking why it does that, it’s because the GPG keys gotta expire at some point, and the images aren’t always updatedBut as #42 talked about, the days of using Debian 8 are counted and Debian Ports should be a new base in the futureLe 18 avr. 2023 à 01:13, Juan Sebastian Muñoz ***@***.***> a écrit :﻿
This is great and all but unfortunately as of now Apr 17th 2023 its complaining about GPG errors
W: GPG error: http://archive.debian.org jessie Release: The following signatures were invalid: KEYEXPIRED 1587841717

I tried this https://unix.stackexchange.com/questions/598344/debian-8-jessie-keyexpired-1587841717 and no luck.
Changed the /etc/apt/sources.list to deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main (https://unix.stackexchange.com/questions/508724/failed-to-fetch-jessie-backports-repository)
I even tried to do the apt-get update with these args apt-get -oAcquire::Check-Valid-Until=no -oAcquire::AllowInsecureRepositories=true update and still nope
Has anyone else stumbled upon this?, I have got several key not found and fixed it with this https://chrisjean.com/fix-apt-get-update-the-following-signatures-couldnt-be-verified-because-the-public-key-is-not-available/
Any thoughts wrt this nowadays 2023 :/?, I am not expecting to get the latest versions of everything since this is for PowerPC architecture and PowerPC was supported until Debian 8, but I would expect to at least be able to install certain packages like emacs and others to make this OS usable :(

**Comment by naruse** [comment]
you are correct I was reading the other issue (#42) and seems the best approach is to get a project with ports.debian.com since this one is just unusable as for now. Thanks!
